integration-test/http/http_system_test.c

```diff
@@ -360,3 +453,58 @@ void test_HTTP_PUT_Request( void )
                      HTTP_METHOD_PUT,
                      PUT_PATH );
 }
+
+/**
+ * @brief Receive a Transfer-Encoding chunked response.
+ */
+void test_HTTP_Chunked_Response( void )
+{
+    sendHttpRequest( &transportInterface,
+                     HTTP_METHOD_GET,
+                     GET_CHUNKED_PATH );
+
+    /* Verify the response is of the length configured, which means the
+     * chunk header was overwritten. */
+    TEST_ASSERT_EQUAL( CHUNKED_BODY_LENGTH, response.bodyLen );
+}
+
+/**
+ * @brief Test how http-parser responds with a response of line-feeds only and
+ * no carriage returns.
+ */
+void test_HTTP_LineFeedOnly_Response( void )
+{
+    HTTPResponse_t response = { 0 };
```

Comment:
Since this is now a static variable up top, is this line necessary?

Reply:
I missed this thanks